### DISCUSSION TITLE
The JSBSim not available for android ?

### DISCUSSION THREAD

**Discussion by YahyaNawir** [JSBSim-Team/jsbsim#1009] [Support requests]
I am currently developing flight simulator using JSBSim plugin in unreal engine 5 for windows, and it is working great, however my client wanted me to package it in apk format for oculus quest 2 installation. is this not possible yet ? I got some error when trying to package for apk format
![Screenshot_1](https://github.com/JSBSim-Team/jsbsim/assets/64287066/2a4935f9-2fba-4ef7-846a-d51d5dfb2545)

**Comment by bcoconni** [comment]
@demonixis has investigated this topic so he might be able to provide some hints.

**Reply by demonixis** [reply]
I'm stuck here too..
